src/accounts/page.rs

```diff
@@ -13,7 +13,7 @@ use leptos_router::create_query_signal;
 #[component]
 pub fn AccountsPage() -> impl IntoView {
     view! {
-        <Title text="Accounts | Search For Mina Accounts"/>
+        <Title text="Accounts | Search For MINA Accounts"/>
```

Comment:
I think that this is wrong. It should indeed be "search of accounts on the Mina blockchain".

Reply:
"Mina" is the name of the blockchain and protocol, like "Bitcoin" or "Ethereum".
"MINA" is the name of the token, like "BTC" or "ETH".